### ISSUE TITLE
[USERSNAP] Forum: Viewing a thread there are no post numbers alongside posts

### ISSUE THREAD

**Issue by dmtrjsg** [Joystream/pioneer#2437]
**Taken at**: https://pioneer-2.vercel.app/#/forum/thread/5
**Fields**:
- Houston, we have a problem.: Forum: Viewing a thread there are no post numbers alongside posts
- Give us some details: Without a post number besides each post, it is difficult to follow very long threads. Especially with pagination this would be problematic.

NOTE: This is NOT the actual `postID` from the chain. The included screencap should show the posts as #1, #2, #3, #4

It could be the chain&#39;s `postID` but this would be a bit of an uncommon way to do it.
- Without a post number besides each post, it is difficult to follow very long threads. Especially with pagination this would be problematic.

NOTE: This is NOT the actual `postID` from the chain. The included screencap should show the posts as #1, #2, #3, #4

It could be the chain&#39;s `postID` but this would be a bit of an uncommon way to do it.
- Discrod Handle: tomato#9464

![SCREENSHOT](https://resources.usersnap.com/company/f9f719df-a929-4461-bb6e-46dbfbceffb7/datapoint_screenshot/5067f47f-f443-4e65-8b0d-b3e647c4c76c-annotated_0a3a526e-649f-4f9a-82f1-e4665d28d951.png?etag=c60f8cbbc3aa9ad067b4d9b557c870d8)
*[Download original image](https://resources.usersnap.com/company/f9f719df-a929-4461-bb6e-46dbfbceffb7/datapoint_screenshot/5067f47f-f443-4e65-8b0d-b3e647c4c76c-annotated_0a3a526e-649f-4f9a-82f1-e4665d28d951.png?etag=c60f8cbbc3aa9ad067b4d9b557c870d8)*

**Browser**: Chrome 98 (Windows 10)
**Screen size**: 3440x1440
**Browser size**: 1718x1306
**[Open #64 in Usersnap](https://app.usersnap.com/l/feedback/9e80be11-490c-4053-90db-36880e327bc8?utm_medium=integration&utm_campaign=integration&utm_content=open)**

Powered by **[Usersnap](https://usersnap.com/?utm_source=product&amp;utm_medium=poweredbylink&amp;utm_campaign=github_entry)**.

**Comment by dmtrjsg** [comment]
think ab it and perhaps some competitive benchmarking may help..

**Comment by bedeho** [comment]
# Context

Replying to second half of this: https://github.com/Joystream/pioneer/issues/2952#issuecomment-1121115183

# Reply

- I don't believe this is actually a well defined problem. What does it mean that its "hard to follow a long thread" in the sense that having post numbering would help. This is totally unclear to me, so it's not really possible to evaluate the proposal as some sort of solution to this. Post numbering is certainly useful, but I think something more specific is being invoked.
- The proposal being described is a major change to how the forum actually works, turning into a threaded discussion, not a linear forum post. This would be an enourmously costly change at multiple levels which would be totally unworkable in an ROI sense, even if it actually solved the vague problem description.

Note: I am a bit concerned about the prioritization here @dmtrjsg, there are more basic usabiility issues with the forum I hope we can tackle, the fact that one can hardly read the actual posts, and it seems you guys need to be better synched up-front on the scale of investment which is justified on tackling many of these long-tail problems.

**Comment by dmtrjsg** [comment]
@bedeho thank you for your feedback! 

I take your point on patching the quick wins, and this is broadly the direction we've been pursuing. With The forum topic, at the time of reviewing the Forum Post visibility it was my initiative to try and address all reported problems holistically.

Let us take your feedback onboard and come back with a version that is skimming quick wins and ensures focus on most relevant parts of content and presenting it better.
